reports/m1_proposal.md

```diff
@@ -0,0 +1,21 @@
+**Description of the data**
+
+The original raw data is downloaded from this URL <https://thepulseofthenation.com/#future>. This dataset is the pull result from Card Against Humanity to the nationally representative of American public about popular social concerns during time period August 1st to 15th 2018. The survey was conducted via face to face, cell phone, and land lines, and respondents' demographic info and opinion choices on survey questions were collected.
+
+The raw dataset contains 800 x 24 datapoints, including collected answers from 800 respondents about 7 demographical questions and 14 opinion questions. It contains 11% of missing data points mainly from redundant 'other' columns.
+
+In order to better understand public opinion of different demographic groups, we have done some initial data preprocessing and data engineering to get prepared for visualization. For data preprocessing, we dropped 5 irrelevant, or vacant columns. We renamed column title into brief phrase form. We added the 'political_spectrum_score' column based on numerical conversion of the 'ideaology' choice to help understand political spectrum. Further, we added the 'time_answered' column to reflect this survey period for dashboard sustainability purpose if there are more incoming surveies in future month.
```

Comment:
maybe let's reference the public opinion is with regards to ... the 2016 Elections? If not here maybe in the beginning paragraph (optional)

Reply:
I feel it may not be fully accurate to say its regard to 2016 Election, a couple of the questions are about future aspects. So saying its about 2016 election may be limiting the scope?

Reply:
agreed